src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php

```diff
@@ -56,13 +56,23 @@ class DbalSessionHandler implements \SessionHandlerInterface
     /**
      * Constructor.
      *
+     * List of available options:
+     *  * db_id_col: The column where to store the session id [default: sess_id]
+     *  * db_data_col: The column where to store the session data [default: sess_data]
+     *  * db_time_col: The column where to store the timestamp [default: sess_time]
+     *
      * @param Connection $con       A connection
      * @param string     $tableName Table name
+     * @param array      $options   An associative array of options
      */
-    public function __construct(Connection $con, $tableName = 'sessions')
+    public function __construct(Connection $con, $tableName = 'sessions', array $options = array())
     {
         $this->con = $con;
         $this->table = $tableName;
+
+        $this->idCol = isset($options['db_id_col']) ? $options['db_id_col'] : $this->idCol;
```

Comment:
That sounds like a good idea. I will update the PR with those changes and be sure to include unit tests within that same commit.